src/puppetlabs/puppetdb/export.clj

```diff
@@ -27,22 +27,25 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Utility Functions
 
+(def file-suffix ".json")
+
 (defn export-filename
   "For anything we store historically and need unique names"
   [{:keys [certname producer_timestamp]}]
   (let [formatted-start-time (->> producer_timestamp
                                   time-coerce/to-date-time
                                   (time-fmt/unparse (time-fmt/formatter "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")))]
```

Comment:
since we don't really care about the string formatting we can probably just call generic-identity-hash on producer_timestamp directly.